src/plugins/workspace.ts

```diff
@@ -148,6 +158,20 @@ export abstract class WorkspacePlugin<T> extends ManifestPlugin {
     );
     newCandidates = await mergePlugin.run(newCandidates);
 
+    if (newCandidates.length === 1) {
```

Comment:
I'm not quite following the issue that this protects against?

Reply:
A plugin's interface is that it post processes an array of candidate pull requests and returns an array of candidate pull requests. It's possible that the plugin returns more than one, but in this case with the `Merge` plugin, we expect a single one.

Reply:
Non blocking, I think if you added a few targeted comments that described the algorithm, it would help me reason my way through the need for this check.